app/addons/documents/__tests__/resources.test.js

```diff
@@ -82,6 +83,25 @@ describe('AllDocs', () => {
   });
 });
 
+describe('NewDoc', () => {
+
+  let getUUID;
+  beforeEach(() => {
+    getUUID = sinon.stub(Helpers, 'getUUID').resolves({ uuids: ['abc9876'] });
+  });
+
+  afterEach(() => {
+    getUUID.restore();
+  });
+
+  it('adds partition key to auto-generated ID', () => {
+    const newDoc = new Models.NewDoc(null, { database: {}, defaultPartitionKey: 'part_key' });
```

Comment:
I guess it was a poor choice of words. I wanted to add some meaning to it instead of simply calling it `partitionKey`, since it's only used as an initial suggestion, and the user can change the final ID, including part key, when they edit the doc.

I will change it to `partitionKey` to avoid confusing